jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -231,7 +232,7 @@ public Videobridge(
      */
     public @NotNull Conference createConference(EntityBareJid name)
     {
-        return createConference(name, Conference.GID_NOT_SET, null);
+        return createConference(name, Conference.GID_NOT_SET, null, false);
```

Comment:
Looks like this API is only exposed for testing. Can you add a comment to the effect of "we default to rtcstatsEnabled=false because this is only used for testing"? I'm wary of us later introducing a bug where rtcstatsEnabled=false because we didn't pay attention to the default.